client/core/trade.go

```diff
@@ -628,6 +630,34 @@ func (t *trackedTrade) activeMatches() []*matchTracker {
 	return actives
 }
 
+// unspentContractAmounts returns the total amount locked in unspent swaps.
+func (t *trackedTrade) unspentContractAmounts(assetID uint32) (amount uint64) {
+	if t.wallets.fromAsset.ID != assetID {
+		// Only swaps sent from the specified assetID should count.
+		return 0
+	}
+	t.mtx.RLock()
+	defer t.mtx.RUnlock()
+	for _, match := range t.matches {
+		side, status := match.Match.Side, match.Match.Status
+		if status >= order.MakerRedeemed || len(match.MetaData.Proof.RefundCoin) != 0 {
+			// Any redemption or own refund implies our swap is spent.
+			// Even if we're Maker and our swap has not been redeemed
+			// by Taker, we should consider it spent.
```

Comment:
"we should consider it spent" Because we are about to redeem it?

Reply:
Err, because taker should "soon" redeem it and we would no longer attempt a refund. If we continue to include it in the swap locked balance, maker's total balance would be temporarily higher than it should be until the swap is redeemed by taker.